docs/core/prerequisites.md

```diff
@@ -0,0 +1,453 @@
+---
```

Comment:
where in the TOC are you putting this? We need a node that will include this, the set up your dev machine, the new article for setting up loc IntelliSense, etc. So probably it would be best to remove this article from the core root.

Reply:
What do you think about putting it (physically) in `docs/core/setup/` where other articles would go too?

Reply:
I like that!